src/wizard/internal/analytics.tsx

```diff
@@ -16,7 +16,10 @@ const createEventDetail = (stepIndex = 0) => `step${stepIndex + 1}`;
 // A custom time cache is used to not clear the timer between navigation attempts
 // This allows us the ability to track time to attempt each step as well as the time to complete
 // each step
-const timeCache: Record<string, number> = {};
+let timeCache: Record<string, number>;
+let currentStepIndex: number;
+let success: boolean;
```

Comment:
Not a big fan of these global vars.

Fine as a temporary solution, but probably should be scoped to  unique `funnelId`

Reply:
Agreed that it's not great and this will hopefully change in the long term.